jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   roots: ["<rootDir>"],
   clearMocks: true,
-  testMatch: ["**/__tests__/**/?(*.)+(spec|test).+(ts|tsx|js)"],
+  testMatch: ["**/(__tests__|src)/**/?(*.)+(spec|test).+(ts|tsx|js)"],
```

Comment:
Does this mean we now have a mish-mash of tests in both the `___tests__` directory and the `src` directory...?

Reply:
That's correct: since one of the test file did not exist, I added it along the source code rather than in the `__tests__` folder.